### ISSUE TITLE
VoiceSwapSample.ipynb [.numpy() bug]

### ISSUE THREAD

**Issue by fr0zenshard** [NVIDIA/NeMo#2561]
You need to call ```.detach()``` before calling ```.numpy()``` in a ```text_to_audio()``` function.

Edited:

```
def text_to_audio(text):
  parsed = spectrogram_generator.parse(text)
  spectrogram = spectrogram_generator.generate_spectrogram(tokens=parsed)
  audio = vocoder.convert_spectrogram_to_audio(spec=spectrogram)
  return audio.to('cpu').detach().numpy()
```

**Comment by blisc** [comment]
You are doing detach() prior to numpy(). It is done from left to right.
